### DISCUSSION TITLE
Trying to run a new project doesn't work

### DISCUSSION THREAD

**Discussion by thany** [microsoft/react-native-windows#12894] [Q&A] [answered]
First of all I'm not happy with the fact that it demands to install 15GB of prerequisites, among which is a full Visual Studio instance. I believe a project should never need to require anything to be installed *globally*, other than the basic tools. If it needs a compiler, why can't it download a compiler and install it in a *local* dependencies directory?

Secondly, it also wants to install Yarn, for reasons that remain unexplained. I'm using NPM and I have no intention of switching to yarn just because a getting started guide tells me to. NPM work perfectly fine, and it's the default package for Node.js, so everyone running Node.js should already have it installed.

Lastly, and this is the winner, after installing all those global dependencies, it still runs into this error:

```
C:\Projects\reactNativeThing>npx react-native run-windows
 MSBuild version 17.9.8+b34f75857 for .NET Framework
Build started 29/03/2024 15:04:32.

Project "C:\Projects\reactNativeThing\windows\reactNativeThing.sln" on node 1 (Restore target(s)).
ValidateSolutionConfiguration:
  Building solution configuration "Debug|x64".
_GetAllRestoreProjectPathItems:
  Determining projects to restore...
Restore:
  X.509 certificate chain validation will use the default trust store selected by .NET for code signing.
  X.509 certificate chain validation will use the default trust store selected by .NET for timestamping.
Done Building Project "C:\Projects\reactNativeThing\windows\reactNativeThing.sln" (Restore target(s)).


Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.96
√ Restoring NuGet packages
 √ Auto-linking...
Success: No auto-linking changes necessary. (7ms)
 √ Found Solution: C:\Projects\reactNativeThing\windows\reactNativeThing.sln
 i Build configuration: Debug
 i Build platform: x64
 × Building Solution: C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Micros...
 × Build failed with message 1>C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.targets(169,5): error : Unable to load the service index for source https://pkgs.dev.azure.com/weareyou/Renewi/_packaging/Renewi/nuget/v3/index.json. [C:\Projects\reactNativeThing\windows\reactNativeThing.sln]. Check your build configuration.
Command failed. Re-run the command with --logging for more information.
```

Mind you, I've just followed the official guide: https://microsoft.github.io/react-native-windows/docs/getting-started

So it's that nuget error. Why is it concerning itself with our corporate nuget repository? What does it want there? It's for a completely separate and unrelated project. I don't like that it's trying to probe into it. And if it's not actually needed, that's fine, but then why does it care about being unable to load the service index there?

With all these problems in mind, this whole project feels extremely beta to me. It's only building apps for Windows, and UWP at that iinm, and with those prerequisites it requires a Windows box as well.

Can someone please clarify what's going on?

**Comment by jonthysell** [comment, marked as answer]
It looks like your build is failing because you've set up some ADO NuGet feed it can't see:

```
C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.targets(169,5): error : Unable to load the service index for source https://pkgs.dev.azure.com/weareyou/Renewi/_packaging/Renewi/nuget/v3/index.json. [C:\Projects\reactNativeThing\windows\reactNativeThing.sln
```

RNW doesn't need that feed, either you're trying to get a NuGet module in your build that comes form that feed, or you've set up a nuget config somewhere to add that feed globally. If you try building again with `npx react-native run-windows --logging` you should get better debug output, including what nuget config files you're using, to track down where that feed is being requested.
